conf/Readme.md

```diff
@@ -0,0 +1,44 @@
+# Configurazione di un PSP
+
+La configurazione di un PSP all'interno della piattaforma pagoPA avviene tramite la trasmissione di:
+
+- Il CatalogoDatiInformativi ( CDI) , file xml il cui schema è rappresentato dal file CatalogoDatiInformativi.xsd
+- Report della configurazione, file html dove vengono riassunte le iformazioni essenziali dei servizi configurati
+
+## building
+
+all'interno della directory `conf` eseguire i comandi
+
+`xslproc -o ./out/CatalogoDatiInformativi.html ./xs3p/xs3p.xsl CatalogoDatiInformativi.xsd`
```

Comment:
if someone try typing `xslproc` it receives as response "command not found".
Into README file add section "Prerequisites" where add all dependecies or packages necessary to run and build.

Furthermore it seems missing some files, example folder `xs3p` is empty